docs/cel2/operators/run-node.md

```diff
@@ -87,13 +87,6 @@ The following sections contain all infromation required to set up your node from
 
 #### Baklava
 
-:::note
-
-These urls will be populated with the assets when they become available, which will be
-shortly after the [last L1 block](https://celo-baklava.blockscout.com/block/countdown/28308599)
```

Comment:
The migrated datadir is not available yet

Reply:
Let's wait with the PR then.